### ISSUE TITLE
Validation does not finish

### ISSUE THREAD

**Issue by offmindby** [postgrespro/pg_probackup#16]
Hi. We have tried to make a full backup from PGDATA directory containing .sock file and got an error 

```
INFO: validate: OOK37F backup and archive log files by CRC
ERROR: invalid type '?' found in "/tmp/probackup/backups/OOK37F/file_database.txt"

```

Here are some lines from file_database.txt

```
base/12371/2336_vm f 0 0 0600 2017-04-17 14:33:50 18446744073709551615 0
base/12376/826_vm f 0 0 0600 2017-04-17 14:33:50 18446744073709551615 0
base/12376/826 f 0 0 0600 2017-04-17 14:33:50 18446744073709551615 0
.s.PGSQL.5432 ? 0 0 0777 2017-04-17 14:33:51 18446744073709551615 0
base/12376/3598_vm f 0 0 0600 2017-04-17 14:33:50 18446744073709551615 0
base/1/12209 f 0 0 0600 2017-04-17 14:33:50 18446744073709551615 0
base/12376/3598 f 0 0 0600 2017-04-17 14:33:50 18446744073709551615 0
```

Looks like pg_probackup fails validation in case postgresql socket file exists in PGDATA directory

**Comment by za-arthur** [comment]
Thank you for the issue! Will check it.

**Comment by gsmol** [comment]
This bug is fixed since v1.1.7.
Sorry for forgetting to inform you about it.
